kubernetes/puppetboard.yml.erb

```diff
@@ -16,7 +16,7 @@ metadata:
   labels:
     app: puppetboard
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
we don't need three replicas for this

Reply:
currently (on marathon) we're running three replicas: using only 1 would be a change.